extensions/2.0/Vendor/MSFT_RigidBodies/schema/node.MSFT_RigidBodies.joint.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "MSFT_RigidBodies glTF extension",
+    "type" : "object",
+    "description" : "Parameters describing how the relative motion of a pair of nodes is constrained.",
+    "allOf" : [ { "$ref" : "glTFProperty.schema.json" } ],
+    "properties" : {
+        "connectedNode" : {
+            "allOf": [{"$ref": "glTFid.schema.json"}],
+            "description": "The ID of the node to which this is connected. If not set then the node will be connected to a fixed point in space."
+        },
+        "jointLimits" : {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
+            "description": "The index of the joint limits in the top level physicsJointLimits array."
+        },
+        "enableCollision" : {
```

Comment:
Is `enableCollision` strictly necessary, or can enabling/disabling collision already be achieved with `collideWithSystems` and `notCollideWithSystems`? I worry about mechanisms for enabling/disabling collisions spread across multiple extensions/objects that may contradict each other.

Reply:
While it's _possible_ to disable this collision by using a collision filter, this helps to avoid having to make a huge number of unique systems. Physics engines have this option as it's frequently desirable to author constraint systems where the constrained bodies overlap (please excuse programmer-art again):

![ConstraintMatchingDemo](https://user-images.githubusercontent.com/1676295/220130774-22e6a030-7f7c-417a-b0da-ad7eefc59e61.png)

but allowing those objects to collide is undesirable from a behavioral perspective, as the contacts and joint are in direct conflict with each other. USD Physics also has the same option on the joint: https://graphics.pixar.com/usd/release/api/class_usd_physics_joint.html#afe0e317d588666298c89eb9b367162ed

Reply:
[Godot has the same setting.](https://docs.godotengine.org/en/latest/classes/class_joint3d.html#class-joint3d-property-exclude-nodes-from-collision) I agree that this good to have, the only reason OMI doesn't have it yet is because we want to make joints their own spec (haven't done this yet, but will do in the coming months).